src/libraries/Microsoft.Extensions.HostFactoryResolver/tests/BuildWebHostInvalidSignature/BuildWebHostInvalidSignature.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
NIT: should we add Directory.Build.props/targets files on the directories of these apps so that they don't import our repo ones? Perhaps to avoid duplication we can move all these test projects under a subfolder under tests, and then just add empty Directory.Build.props/targets there.

Reply:
I didn't want to do that as it would disrupt the output paths.  I used the same pattern we follow elsewhere for test helper libs.

Reply:
Here's where I copied the pattern from: https://github.com/dotnet/runtime/tree/master/src/libraries/System.Runtime.Extensions/tests